pkg/api/status/v1.go

```diff
@@ -84,43 +86,30 @@ func (sapi *StatusAPI) Register(r *route.Router, tracer opentracing.Tracer, logg
 }
 
 func (sapi *StatusAPI) httpServeStats(r *http.Request) (interface{}, []error, *api.ApiError) {
-	s, err := sapi.getTSDBStats(r, labels.MetricName)
+	stats, err := sapi.getTSDBStats(r, labels.MetricName)
 	if err != nil {
-		return nil, nil, &api.ApiError{Typ: api.ErrorInternal, Err: err}
+		return nil, nil, err
 	}
 
-	if s == nil {
+	if stats == nil {
 		return nil, nil, &api.ApiError{Typ: api.ErrorBadData, Err: fmt.Errorf("unknown tenant")}
 	}
 
-	metrics, err := sapi.options.Registry.Gather()
-	if err != nil {
-		return nil, []error{err}, nil
-	}
-
-	chunkCount := int64(math.NaN())
```

Comment:
This is an extremely hacky way to get number of head chunks and I suggest we remove the field from the response. Admins can still get this data since it is exposed as a metric with separate series per tenant.

I'm not sure why Prometheus does it this way tbh.

Reply:
For consistency I guess. So you propose to skip this in our API?

Reply:
I would still do it IMO

Reply:
So we will split the values per tenant using the tenant label?